CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+#
+# CMAKE SETUP (version, build style)
+#
+cmake_minimum_required (VERSION 2.8 FATAL_ERROR)
+project ("Quick-DER" C)
+
+set (CMAKE_MACOSX_RPATH 0)  # Don't use rpaths (but we don't build executables anyway)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
+
+include(MacroEnsureOutOfSourceBuild)
+macro_ensure_out_of_source_build("Do not build Quick-DER in the source directory.")
```

Comment:
Ik vind het bouwen buiten de source om elegant, maar waarom verplichten?
[Als het is om de Makefiles tegen overschrijven te beschermen... denk ik, wie zijn billen brandt die zit maar wat minder comfy]